### PR TITLE
Update to list

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Change to list

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 1]
```diff
@@ -338,29 +338,20 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/DocumentTypes"
-  "/lookup/courtInformation/{courtName}":
+  "/lookup/courtLocations/":
```
Comment (alexjoybc) [inline src/backend/efiling-api/jag-efiling-api.yaml:341]: I would just go with "/court" court will be the resource

Comment (alexjoybc) [reply src/backend/efiling-api/jag-efiling-api.yaml:341]: and add a querystring parameter ?level=P / S

**Review by alexjoybc — CHANGES_REQUESTED** [review]

File: src/backend/efiling-api/jag-efiling-api.yaml (hunk 1 above)
Comment (TayGov) [reply src/backend/efiling-api/jag-efiling-api.yaml]: Updated

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 2]
```diff
@@ -338,18 +340,18 @@ paths:
...
+  "/courts":
     get:
-      summary: Get court information
-      operationId: GetCourtInformation
+      summary: Get court locations
+      operationId: GetCourtLocations
       description: get court information by court name
       tags:
-        - lookup
+        - court
```
Comment (alexjoybc) [inline src/backend/efiling-api/jag-efiling-api.yaml:349]: can you add client endpoints

**Review by alexjoybc — CHANGES_REQUESTED** [review]

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**